client/my-sites/domain-tip/index.jsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import noop from 'lodash/noop';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { abtest } from 'lib/abtest';
+import { getSite, getSiteSlug } from 'state/sites/selectors';
+import { getDomainsSuggestions, } from 'state/domains/suggestions/selectors';
+import QueryDomainsSuggestions from 'components/data/query-domains-suggestions';
+import UpgradeNudge from 'my-sites/upgrade-nudge';
+
+function getQueryObject( site, siteSlug ) {
+	if ( ! site || ! siteSlug ) {
+		return null;
+	}
+	return {
+		query: siteSlug.split( '.' )[ 0 ],
+		quantity: 1,
+		vendor: abtest( 'domainSuggestionVendor' )
+	};
+}
+
+const DomainTip = React.createClass( {
+
+	propTypes: {
+		className: React.PropTypes.string,
+		siteId: React.PropTypes.number.isRequired
+	},
+
+	getDefaultProps() {
+		return {
+			quantity: noop
+		};
+	},
+
+	shouldDisplay() {
+		return true;
```

Comment:
This will be temporary until we hash out which conditions we want a domain tip to display.